source/upgrade.txt

```diff
@@ -145,7 +145,16 @@ The v4.1 driver introduces the following breaking changes to the ABI:
 
   .. note::
 
-     This does not affect users who compile with C++17 or newer and have not set ``BSONCXX_POLY_USE_IMPLS=ON``
+     This does not affect users who compile with C++17 or newer and have not set
+     ``BSONCXX_POLY_USE_IMPLS=ON``
+
+.. _version-4.1-enable-tests-config:
+
+- Sets the ``ENABLE_TESTS`` configuration option to ``OFF`` by default,
+  including for auto-downloaded versions of the {+driver-short+}. To
+  re-enable building test targets, you must set ``ENABLE_TESTS`` to ``ON``.
+  To include test targets in the "all" target, you must also set the
+  ``BUILD_TESTING`` option to ``ON``.
```

Comment:
S: If you do leave this as an entry in the 4.0 section, you can reword here to something like:
```suggestion
- Fixes a bug where auto-downloaded versions of the C driver set the ``ENABLE_TESTS`` configuration option to ``ON``, overriding the {+driver-short+} default value. ``ENABLE_TESTS`` is now set to ``OFF`` by default in all cases.
```

Reply:
Also, if this is not a breaking change I'd just add it to What's New and not Upgrade